src/lib/dashboard-model.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const config = require('../config');
+const query = config.postgres.mock ? require('../../mock/mock-query') : require('./db/query');
+
+const build = () => Promise.join(
+  query.searchTotals(true),
+  query.searchTotals(false),
+  (allTime, todaySearches) => ({
+    allTime,
+    todaySearches,
+  })
```

Comment:
Worth condensing queries or at least doing them as the same transaction(pgpromise has another concept for this but I can't remember it).

Reply:
Something for another day though!

Reply:
agreed